libraries/AP_NavEKF3/AP_NavEKF3.cpp

```diff
@@ -604,6 +604,50 @@ const AP_Param::GroupInfo NavEKF3::var_info[] = {
     // @Units: mGauss
     AP_GROUPINFO("MAG_EF_LIM", 56, NavEKF3, _mag_ef_limit, 50),
 
+    // @Param: EAS_DEFAULT
+    // @DisplayName: Default airspeed assumption
+    // @Description: Default value of equivalent airspeed assumed if an airspeed value is required but not available. Set to cruise speed.
+    // @Range: 10.0 100.0
+    // @Increment: 0.5
+    // @User: Advanced
+    // @Units: m/s
+    AP_GROUPINFO("EAS_DEFAULT", 57, NavEKF3, _easDefault, 15.0f),
```

Comment:
I think this should be supplied by a call from the vehicle code

Reply:
yes, could check the trim airspeed

Reply:
Done